mmv1/third_party/terraform/services/iamworkforcepool/resource_iam_workforce_pool_provider_scim_token_test.go

```diff
@@ -0,0 +1,169 @@
+package iamworkforcepool_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccIAMWorkforcePoolWorkforcePoolProviderScimToken_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckIAMWorkforcePoolWorkforcePoolProviderScimTokenDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIAMWorkforcePoolWorkforcePoolProviderScimToken_full(context),
+			},
+			{
+				ResourceName:            "google_iam_workforce_pool_provider_scim_token.scim_token",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"state"},
+			},
+			{
+				Config: testAccIAMWorkforcePoolWorkforcePoolProviderScimToken_update(context),
```

Comment:
update tests need ConfigPlanChecks to verify that fields are updated in-place. https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test


```suggestion
				Config: testAccIAMWorkforcePoolWorkforcePoolProviderScimToken_update(context),
				ConfigPlanChecks: resource.ConfigPlanChecks{
					PreApply: []plancheck.PlanCheck{
						plancheck.ExpectResourceAction("google_iam_workforce_pool_provider_scim_token.scim_token", plancheck.ResourceActionUpdate),
					},
				},
```